bee-pow/tests/score.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_pow::score::compute_pow_score;
+
+#[test]
+fn score() {
+    let message: [u8; 21] = [
+        0x48, 0x65, 0x6c, 0x6c, 0x6f, 0x2c, 0x20, 0x57, 0x6f, 0x72, 0x6c, 0x64, 0x21, 0x5e, 0xe6, 0xaa, 0xaa, 0xaa,
+        0xaa, 0xaa, 0xaa,
+    ];
+
+    assert!((compute_pow_score(&message) - 937.2857142857143).abs() < f64::EPSILON);
```

Comment:
Maybe a comment here about where the magic number comes from – it's not immediately obvious (to me at least!)

Reply:
That's just random for the tests, but I'll add links to where I took the tests from.

Reply:
Yep, makes sense after following the link!